universum/modules/launcher.py

```diff
@@ -308,18 +308,18 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--config", "-cfg", dest="config_path", metavar="CONFIG_PATH",
                             help="Path to project config file (example: -cfg=my/prject/my_conf.py). "
-                                 "Mandatory parameter.")
+                                 "Mandatory parameter")
 
-        parser.add_argument("--filter", "-f", dest="step_filter", action='append',
+        parser.add_argument("--filter", "-f", dest="step_filter", action='append', metavar="STEP_FILTER",
                             help="Filter steps to execute. A single filter or a set of filters separated by ':'. "
-                                 "Exlude using '!' symbol before filter. "
+                                 "Exclude using '!' symbol before filter. "
                                  "Example: -f='str1:!not str2' OR -f='str1' -f='!not str2'. "
-                                 "See online docs for more details.")
+                                 "See online documentation for more details")
```

Comment:
Can we give direct link to https://universum.readthedocs.io/en/latest/args.html#Configuration%20execution ?